### DISCUSSION TITLE
gnome-keyring-daemon missing on fat clients

### DISCUSSION THREAD

**Discussion by X-Dimension-88MPH** [ltsp/ltsp#852] [Q&A]
We are using LTSP on Ubuntu 22.04LTS and Gnome 3 in chrootless mode and want to allow our users to save passwords for smb shares in Nautilus/Gnome-Keyring. 
But after some testing we found that it's impossible to do so, because on the fat clients /usr/bin/gnome-keyring-daemon does not exist, although it is installed on the server.

Is there a good reason for that, or is it a bug? Am i the only one where gnome-keyring-daemon is missing on the clients?

**Comment by alkisg** [comment]
See https://github.com/ltsp/ltsp/issues/22 and https://github.com/ltsp/ltsp/discussions/419.
If you have any more questions about the issue, let's continue there.
